spago.lock

```diff
@@ -283,7 +283,7 @@ workspace:
         - unicode
         - versions
       git: https://github.com/JordanMartinez/purescript-language-purescript.git
-      ref: v0.1.1
+      ref: db4377dea03ba9c5273a93a8368a53f1d87882c1
```

Comment:
Necessary patch until we get a new release:
https://github.com/JordanMartinez/purescript-language-purescript/commit/db4377dea03ba9c5273a93a8368a53f1d87882c1